app/src/atoms/AtomLayout.re

```diff
@@ -0,0 +1,21 @@
+module Styles = {
+  open Css;
+
+  let left = style([flex(`num(1.0)), display(`flex)]);
```

Comment:
Totally valid approach, but can we try to use grids wherever possible to stick to one layout principle. Unless there is strong reason to use flex here.

Reply:
My understanding of the "flexbox vs grid" topic is that it's easier to lay out stuff along one axis (1d) with grid and multiple-axes (2d) with flexbox. And that they're commonly used together, as described [here](https://css-tricks.com/css-grid-replace-flexbox/).

Does @juliendonck have an opinion on this?

Reply:
Yeah, how I understood it they are meant to be used interchangeably. Grid for page structure, flexbox for layout.

Reply:
Great, thanks for clarification and pointers!